Data import

samsawan opened the pull request:
Sorry I missed Friday, totally slipped my mind :(

Commit:
importing new data

also doing some formatting on the README

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
importing new data

Commit:
Merge branch 'master' of https://github.com/vsalvino/clevelandtips

Commit:
data import

vsalvino commented:
All good, that's why it's nice to have 2 shifts per day. I'm seeing a couple duplicates in the diff, could you please check that out, and also hand-check a couple of the restaurant names that seem to be people entering multiple restaurant names?

samsawan commented on site/data/data.json:
```diff
@@ -10825,6 +10905,20 @@
     ]
   },
+  {
+    "name": "Joeys Italian Grille (Madison)",
```
Fixing this entry

samsawan commented on site/data/data.json:
```diff
@@ -13648,6 +13822,21 @@
     ]
   },
+  {
+    "name": "Merry Arts, Lakewood Village Tavern, Academy Tavern",
```
@vsalvino is this a suspect entry? I hid it, added carl to the Lakewood village entry

vsalvino replied:
Some people try to list multiple restaurants, even though we ask them not to. I usually just take the first one and chop off the rest.

samsawan commented on site/data/data.json:
```diff
@@ -14804,6 +14993,20 @@
     ]
   },
+  {
+    "name": "Oishi Asian Restaurant",
```
Duplicate, removing

samsawan commented on site/data/data.json:
```diff
@@ -15959,6 +16162,20 @@
     ]
   },
+  {
+    "name": "Parma Pit Stop",
```
Duplicate, removing

samsawan commented on site/data/data.json:
```diff
@@ -19664,6 +19964,21 @@
     ]
   },
+  {
+    "name": "Spotted Owl And L'Abatros",
```
I hid this entry, added Kindra to the Spotted Owl entry

vsalvino replied:
Perfect. You can actually delete it all together, the script won't re-add the same person regardless of what their place name is.

samsawan reviewed (commented):
pushing another commit up shortly, i think this addresses the dupes

Commit:
fixing dupes

Commit:
address another dupe worker entry

Commit:
removing multi entries

vsalvino commented:
Sorry for all the feedback, appreciate the help. The Sunday peeps must have been having a hard time following instructions on the sign-up form 😅

samsawan commented:
No problem @vsalvino, just removed the multi entries as well. I didn't think the import script would recognize the fact that I had deleted them from the CSV and try to re-add the multi entries when it was run again. I was mistaken, so hell yeah, no need to keep that data there with a 'hide' flag. 👍

Commit:
mispelling

vsalvino commented:
Yea, the importing is a mess and super hacky. There definitely has to be a better way... although probably would require making an actual signup page versus a simple google form.
